mithril-explorer/src/components/RegistrationDiscordFormatModal.js

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from "react";
+import { Card, Modal } from "react-bootstrap";
+import CopyButton from "#/CopyButton";
+import { useSelector } from "react-redux";
+import { getSelectedAggregatorPools } from "@/store/poolsSlice";
+
+/**
+ * Modal to show a list of registrations in a discord formatted code block
+ *
+ * @param registrations List of registrations movements
+ * @param onClose Callback to call when the user ask to close the modal
+ * @param mode 'in', 'out' or 'undefined': if undefined, the modal will not show
+ */
+export default function RegistrationDiscordFormatModal({ registrations, onClose, mode }) {
+  const allPools = useSelector((state) => getSelectedAggregatorPools(state));
+  const [textToCopy, setTextToCopy] = useState(undefined);
+
+  useEffect(() => {
+    if (mode === undefined) {
+      return;
+    }
+    let text = "";
+
+    for (const [epoch, movements] of Object.entries(registrations).reverse()) {
+      const signers = mode === "in" ? movements.in : movements.out;
+      if (signers.length === 0) {
+        continue;
+      }
+
+      text += `Since epoch **#${epoch}**:\n`;
+      for (const signer of signers
+        .map((s) => ({
+          party_id: s.party_id,
+          pool_ticker: allPools?.find((p) => p.party_id === s.party_id)?.pool_ticker ?? "",
+        }))
+        .sort(compareSigners)) {
+        text += `* ${signer.party_id}`;
+
+        if (signer.pool_ticker !== "") {
+          text += ` **${signer.pool_ticker}**`;
+        }
+
+        text += `\n`;
+      }
+    }
+
+    setTextToCopy(text);
+  }, [registrations, mode, allPools]);
+
+  function compareSigners(left, right) {
+    // Sort by pool_ticker then party_id
+    return (
+      left.pool_ticker.localeCompare(right.pool_ticker) ||
+      left.party_id.localeCompare(right.party_id)
+    );
+  }
+
+  function handleModalClose() {
+    onClose();
+  }
+
+  return (
+    <Modal
+      show={mode !== undefined}
+      onHide={handleModalClose}
+      size="xl"
+      aria-labelledby="contained-modal-title-vcenter"
+      centered>
+      <Modal.Header closeButton>
+        <Modal.Title>
+          <i className="bi bi-discord"></i> Discord formatted table of{" "}
```

Comment:
Could we make it a bit more generic?
```suggestion
          <i className="bi bi-discord"></i> Markdown formatted table of{" "}
```

Reply:
Oh looks like Discord updated their markdown support some time ago and [now support more tags](https://support.discord.com/hc/en-us/articles/210298617-Markdown-Text-101-Chat-Formatting-Bold-Italic-Underline).
Before it discord only support bold, italic, and crossed text, all other markdown features were ignored. That's why I though that specifying it was discord compatible mattered.

I will do that and add  on the pool ticker the CExplorer link if available,.

Reply:
Well adding the CExplorer link to the pool ticker in the markdown make it really easy to exceed discord message size limit so I won't be able to do that sadly (or we would need to split the message in several parts ... or pay discord nitro).

Reply:
It's fine without the link :+1: 
I forgot that we could reach the message size limit with Discord :man_shrugging: